src/librustc_codegen_llvm/back/link.rs

```diff
@@ -1626,8 +1626,12 @@ fn relevant_lib(sess: &Session, lib: &NativeLibrary) -> bool {
 fn is_full_lto_enabled(sess: &Session) -> bool {
```

Comment:
With the comment below, perhaps this function could be renamed to something like `are_upstream_rust_objects_already_included`? (albeit a bit wordy)